docs/terminal/melbourne.md

```diff
@@ -327,12 +327,13 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
 Refer to [Reclassifications](#mb-ctr) for operations when MB ADC is offline.
 
 #### Departures
-Departures in to ML TCU Class C airspace require a "Next" call, where ML TCU will provide the cleared level. There is no standard assignable level.
```

Comment:
Can we just put in here (and anywhere else similar in this PR) the words "standard assignable level"? Reason being, a lot of people will open up the LIs, and Ctrl+F "standard assignable", or use the search function at the top of the SOPs. This should make it easier to find quicker